source/includes/common-mc-admin-config.rst

```diff
@@ -996,17 +996,6 @@ Specify the URL for the webhook service.
 
 .. end-minio-notify-webhook-endpoint
 
-.. start-minio-notify-webhook-auth-token
```

Comment:
Removed this include because the needed text is slightly different in the two locations it was used.
(Also why the others aren't includes.)